\documentclass[PhD]{PHlab-thesis}

\addbibresource{thesis-example.bib}

\newcommand*\Departmentä¸­æ–‡{è³‡è¨Šå·¥ç¨‹å­¸ç ”ç©¶æ‰€}
\newcommand*\Departmentè‹±æ–‡{Institute of Computer Science and Information Engineering}

\newcommand*\ThesisTitleä¸­æ–‡{è¦–è¦ºåŒ–è½‰éŒ„å› å­çµåˆä½é»DNA\é€€{1}ç”²åŸºåŒ–çš„æ–°æ–¹æ³•ï¼šMethylSeqLogo}
\newcommand*\ThesisTitleè‹±æ–‡{MethylSeqLogo: A Novel Method for the Vizualization of DNA Methylation at Transcription Factor Binding Sites}
\newcommand*\ThesisNoteä¸­æ–‡{ç¤ºä¾‹:å…¶å¯¦å¾ç¿¡æ›¼æ˜¯æ±äº¬å¤§å­¸ç•¢æ¥­çš„åšå£«}% For real thesis omit, or use {åˆç¨¿} etc.
\newcommand*\ThesisNoteè‹±æ–‡{Just an example.  Fei-Man actually graduated from Tokyo Univ.}% For real thesis omit, or use {draft} etc.

\newcommand*\Studentä¸­æ–‡{å¾ç¿¡æ›¼}
\newcommand*\Studentè‹±æ–‡{Fei-man Hsu}

\newcommand*\Advisorä¸­æ–‡{è³€ä¿ç¾…}
\newcommand*\Advisorè‹±æ–‡{Paul Horton}

%% æœæœ‰å…±åŒæŒ‡å°è€å¸«å¯ä»¥ç”¨:
%% \newcommand*\CoAdvisorAä¸­æ–‡{}
%% \newcommand*\CoAdvisorAè‹±æ–‡{}
%% \newcommand*\CoAdvisorBä¸­æ–‡{}
%% \newcommand*\CoAdvisorBè‹±æ–‡{}


\newcommand*\YearMonthè‹±æ–‡{July, 2022}
\newcommand*\YearMonthä¸­æ–‡{ï¼‘ï¼‘ï¼‘å¹´ï¼—æœˆ}

\pagestyle{fancy}% Use fancyhdr
\begin{document}


\newcommand*\Keywordsè‹±æ–‡{bioinformatics, genomics, string algorithms}
\newcommand*\Abstractè‹±æ–‡{%
\textbf{BACKGROUND:}
Sequence logos can effectively visualize position specific base preferences evident in a collection of binding sites of some transcript factor.  But those preferences usually fall far short of fully explaining binding specificity.
Interestingly, several transcription factors bind sites of potentially methylated DNA.  For example, MYC binds CpG sites.  This may increase binding specificity as such sites are 1) highly under-represented in the genome, and 2) offer additional, tissue specific information in the form of hypo- or hyper-methylation.  Fortunately, data suitable to look into this possibility (e.g.\ bisulfite sequencing) is readily available.\\
\textbf{METHOD:}
We developed MethylSeqlogo, an extension of sequence logos which adds DNA methylation information to sequence logos.  MethylSeqlogo includes new elements to indicate DNA methylation and under-represented dimers in each position of a set of aligned binding sites.  Our method displays information from both DNA strands, and takes into account the sequence context (CpG or other) and genome region (promoter versus whole genome) appropriate to properly assess the expected background dimer frequency and level of methylation.\\
When designing MethylSeqlogo, we took care to preserve the usual sequence logo meaning of heights; in which the relative height of nucleotides within a column represents their proportion in the binding sites, while the absolute height of each column represents information (relative entropy) and the height of all columns added together represents total information.\\
\textbf{RESULTS:}
We present several figures illustrating the utility of using MethylSeqlogo to summarize data from CpG binding transcription factors.  The logos show that unmethylated CpG binding sites are a feature of transcription factors such as Myc and ZBTB33, while some other CpG binding transcription factors, such as CEBPB, appear methylation neutral.
We also compare MethylSeqlogo with two previously reported ways to create methylation aware sequence logos and conclude that MethylSeqlogo compares favorably to those methods.
Our freely available software enables users to explore large-scale bisulfite and ChIP sequencing data sets --- and in the process obtain publication quality figures.}


\newcommand*\Keywordsä¸­æ–‡{ç”Ÿå‘½ç§‘å­¸ã€åŸºå› çµ„ã€å­—ä¸²æ¼”ç®—æ³•}
\newcommand*\Abstractä¸­æ–‡{%
MethylSeqLogo...è¡ä¼¸sequence logoçš„è¦–è¦ºåŒ–æ–¹æ³•æ”¹å–„åŒ…æ‹¬DNAç”²åŸºåŒ–çš„è³‡è¨Šã€‚æ ¸åŸºå› é«”èƒå˜§å•¶ç”²åŸºåŒ–ç‚ºè„Šæ¤å‹•ç‰©é‡è¦çš„è¡¨è§€éºå‚³ä¹‹ä¸€ã€‚å¯ä»¥ç¶“éäºç¡«é…¸é¹½è™•ç†ä¹‹å¾Œå®šåºDNAçš„å¯¦é©—(bisulfite sequencing)å¾—åˆ°èƒå˜§å•¶ç”²åŸºåŒ–çš„è³‡è¨Šã€‚
ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚ç”²åŸºåŒ–çš„è³‡è¨Šã€‚
}

\newcommand*\Acknowledgements{%
æ„Ÿè¬æˆ‘...}



\input{frontmatter}% å°é¢é , å£å§”ä¸­è‹±æ–‡ç°½åå–®, èªŒè¬, ä¸­è‹±æ–‡æ‘˜è¦, è«–æ–‡ç›®éŒ„, åœ–è¡¨ç›®éŒ„


%â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€  List of Symbols  â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€
\renewcommand\nomgroup[1]{%
  \item[\bfseries
  \ifstrequal{#1}{A}{General}{%
  \ifstrequal{#1}{Z}{Gene/Protein Names}%
  }]}

\nomenclature[A]{$\lg$}{Logarithm base 2}
\nomenclature[A]{KL\ Divergence}{Kullback-Liebler Divergence}
\nomenclature[Z]{Myc}{MYC proto-oncogene}
\nomenclature[Z]{USF-1}{Upstream stimulatory factor 1}

\printnomenclature[5cm]

\newpage
\setcounter{page}{1}
\pagenumbering{arabic}



\chapter{Introduction}
DNA methylation is a major component of the epigenetic state of cells, as as such plays a critial role in the differentiation and maintenance of cell type.  Consequentially, aberrant DNA methylation plays a central role in cancer --- a manifestation of defective regulation of cell behavior.


\chapter{Related Works}
The characteristics of NuMT insertion sites have been analyzed in depth~\cite{TFH12}.
Add your related works here.

Schneider \& Stephens introduced sequence logos many years ago~\cite{SS90}.



\chapter{Method}
\section{Mathematical Model}
\subsection{Notation}
The notation used in this thesis is summarized in table~\ref{tab:notation}.

\subsection{Likelihood Calculation}
Our model assumes the data follows a standard normal distribution $ğ’©(Î¼,ÏƒÂ²)$, so the likelihood function is:

\begin{align*}
â„’  \;=&\;\;  \frac{1}{Ïƒ\sqrt{2Ï€}}\, \exp\Big( \frac{-(x-Î¼)Â²}{2ÏƒÂ²}\Big)\\[.5ex]% .5ex adds extra vertical height
     âˆ&\;\;  Ïƒâ»Â¹\, \exp\big( -Â½\,zÂ²)  \hspace{1cm} \text{where~~} z â‰ \frac{x-Î¼}{Ïƒ}
\end{align*}



\begin{table}
\begin{tabularx}{0.9\linewidth}{lX}
Notation                                               &  Description\\
\toprule
$P_{\text{meth}\,|\,\textsf{context,BG}}$                    &  The background probability of methylation by context.\\[.3ex]
$n_{\textsf{context},\,i}$                                 &  The number of cytosines (with sufficient read coverage)\newline of the given context occurring at position $i$ in the set of TFBS.\\[.3ex]
$\overline{v}\,|\,\scriptstyle{\textsf{context},i}$  & The average methylation values of those cytosines.\\
\bottomrule
\end{tabularx}
\caption{Summary of notation used in this thesis.}
\label{tab:notation}
\end{table}

\section{Proposed Scheme}
Write a bunch of stuff here.

	
\chapter{Results}
Describe your results here.


\chapter{Discussion}
Discussion the significance or your results.


\section{Future Work}


\chapter{Conclusion}
Add your conclusions here.


\newpage
\AddToContents{Bibliography}
\printbibliography


\end{document}
